### PR TITLE
CFE-473: Add information about profiling data lifetime

### PR DESCRIPTION
**Pull request by xenolinux**
Version(s): 4.12+

Issue: https://issues.redhat.com/browse/CFE-473

Link to docs preview: http://file.rdu.redhat.com/sdudhgao/nob-data-profiling/scalability_and_performance/node-observability-operator.html#running-profiling-query_node-observability-operator

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by alebedev87** [comment]
Thanks for the PR, @xenolinux !

Just a small proposal. `emptyDir` change is just a little technical detail which doesn't change a lot for the end user. I think it doesn't really deserve to be at the very beginning of the NodeObservability Operator's page. I was thinking of [Running the profiling query chapter](https://52213--docspreview.netlify.app/openshift-enterprise/latest/scalability_and_performance/node-observability-operator.html#running-profiling-query_node-observability-operator) more. This phrase can be a good candidate for mentioning `emptyDir`:
```
After the profiling query is complete, you must retrieve the profiling data inside the container file system /run/node-observability directory.
```

File: modules/node-observability-run-profiling-query.adoc [hunk 1]
```diff
@@ -6,7 +6,7 @@
 [id="running-profiling-query_{context}"]
 = Running the profiling query
 
-To run the profiling query, you must create a `NodeObservabilityRun` resource. The profiling query is a blocking operation that fetches CRI-O and Kubelet profiling data for a duration of 30 seconds. After the profiling query is complete, you must retrieve the profiling data inside the container file system `/run/node-observability` directory.
+To run the profiling query, you must create a `NodeObservabilityRun` resource. The profiling query is a blocking operation that fetches CRI-O and Kubelet profiling data for a duration of 30 seconds. After the profiling query is complete, you must retrieve the profiling data in the `emptyDir` volume. You can access the profiling data as long as a pod is in the `running` status.
```
Comment (alebedev87) [inline modules/node-observability-run-profiling-query.adoc:9]:
```suggestion
To run the profiling query, you must create a `NodeObservabilityRun` resource. The profiling query is a blocking operation that fetches CRI-O and Kubelet profiling data for a duration of 30 seconds. After the profiling query is complete, you must retrieve the profiling data inside the container file system `/run/node-observability` directory. The data's life time is bound to the agent pod via the `emptyDir` volume, so you can access the profiling data as long as a pod is in the `running` status.
```

Comment (xenolinux) [reply modules/node-observability-run-profiling-query.adoc]: Addressed

File: modules/node-observability-run-profiling-query.adoc [hunk 2]
```diff
@@ -6,7 +6,7 @@
 [id="running-profiling-query_{context}"]
 = Running the profiling query
 
-To run the profiling query, you must create a `NodeObservabilityRun` resource. The profiling query is a blocking operation that fetches CRI-O and Kubelet profiling data for a duration of 30 seconds. After the profiling query is complete, you must retrieve the profiling data inside the container file system `/run/node-observability` directory.
+To run the profiling query, you must create a `NodeObservabilityRun` resource. The profiling query is a blocking operation that fetches CRI-O and Kubelet profiling data for a duration of 30 seconds. After the profiling query is complete, you must retrieve the profiling data inside the container file system `/run/node-observability` directory. The lifetime of data is bound to the agent pod through the `emptyDir` volume, so you can access the profiling data as long as the agent pod is in the `running` status.
```
Comment (abhatt-rh) [inline modules/node-observability-run-profiling-query.adoc:9]: Refer to ISG > Word Usage for [as long as](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#as-long-as)
```suggestion
To run the profiling query, you must create a `NodeObservabilityRun` resource. The profiling query is a blocking operation that fetches CRI-O and Kubelet profiling data for a duration of 30 seconds. After the profiling query is complete, you must retrieve the profiling data inside the container file system `/run/node-observability` directory. The lifetime of data is bound to the agent pod through the `emptyDir` volume, so you can access the profiling data while the agent pod is in the `running` status.
```

**Review by abhatt-rh — COMMENTED** [review]
LGTM, except a minor comment!

/remove-label peer-review-in-progress
/remove-label peer-review-needed
/label peer-review-done

File: modules/node-observability-run-profiling-query.adoc (hunk 2 above)
Comment (xenolinux) [reply modules/node-observability-run-profiling-query.adoc]: Addressed

**Pull request merged**